lib/shipitron/server/upload_build_cache.rb

```diff
@@ -9,6 +9,7 @@ class UploadBuildCache
       required :application
       required :s3_cache_bucket
       required :build_cache_location
+      required :region
```

Comment:
Is this actually required or should it be defaulted?

Reply:
It's actually required now. The aws cli needs to know the s3 bucket region.

Reply:
Yeah, just wondered why you didn't just use `optional` and default it to the east region.

Reply:
That's not a sane default and we already have the right answer in the config.